tests/integration/Infrastructure/Services/Common/FlysystemScanStorageTest.php

```diff
@@ -73,13 +72,13 @@ public function testGetReturnsCorrectFile() : void
     {
         $contents = Image::fromBlank(1, 1)->toString();
         $filename = 'foo.jpg';
-        $this->storage->save($this->getFilePath($filename), $contents);
+        $filepath = $this->getFilePath($filename);
+        $this->storage->save($filepath, $contents);
 
-        $file = $this->storage->get($this->getFilePath($filename));
+        $file = $this->storage->get($filepath);
 
-        $newFilename = explode('_', $file->getFileName(), 2)[1];
-        $this->assertSame(FilePath::generatePath(self::FILE_PATH_PREFIX, $filename), $file->getPath());
-        $this->assertSame($filename, $newFilename);
+        $this->assertSame($filepath->getPath(), $file->getPath());
+        $this->assertSame($filename, $file->getFileNameWithoutPrefix());
```

Comment:
Tady klidně stačí jednodušší:
```php
$this->assertStringEndsWith($filename, $file->getFilePath());
```

Reply:
to klidně upravím, ale i tak pak budu chtit dělat ten getFileNameWithoutPrefix

Reply:
Není třeba. Tím, že tam ta metoda zůstane, je zbytečný to nějak víc zjednodušovat

Reply:
dal jsem to pryc a přidám to az s tím dalším PR